baseline.yaml

```
@@ -720,6 +720,55 @@ criteria:
       in a separate repository and fetched during
       a specific well-documented pipeline step.
     control_mappings: # TODO
+    security_insights_value: # TODO
+    scorecard_probe: # TODO
+  - id: OSPS-53
+    maturity_level: 2
+    category: Build & Release
+    criteria: |
+      All released software assets MUST be signed
```

This is not nearly rigorous enough. Anyone can sign anything. The goal here is to cryptographically tie some element of the software (e.g. it's build) to the approved maintainers of that software. As written it's way too weak.

---

What about something like
```suggestion
      All released software assets MUST be signed or accounted for in a signed manifest including each asset's cryptographic hashes (such as an SBOM, provenance attestation or similar). The identity of the person or process authoring the software release must be published, discoverable and must be taken into account during the verification process.
```